cypress/.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-env node */
+
+module.exports = {
+  extends: ['../.eslintrc.js', 'plugin:cypress/recommended', 'prettier'],
```

Comment:
Extends our main eslint configuration, extends the recommend cypress config and prettier is last so that it's formatting rules always win.